README.md

```diff
@@ -1,175 +1,149 @@
+<a href="https://jfrog.com">
+    <img src=".github/jfrog-logo-2022.svg" alt="JFrog logo" title="JFrog" align="right" height="50" />
+</a>
+
 # Terraform Provider Artifactory
 
 [![Actions Status](https://github.com/jfrog/terraform-provider-artifactory/workflows/release/badge.svg)](https://github.com/jfrog/terraform-provider-artifactory/actions)
 [![Go Report Card](https://goreportcard.com/badge/github.com/jfrog/terraform-provider-artifactory)](https://goreportcard.com/report/github.com/jfrog/terraform-provider-artifactory)
 
-To use this provider in your Terraform module, follow the documentation [here](https://registry.terraform.io/providers/jfrog/artifactory/latest/docs).
+## Releases
 
-## License requirements:
+Current major release: **6.x**
 
-This provider requires access to Artifactory APIs, which are only available in the _licensed_ pro and enterprise editions. You can determine which license you have by accessing the following URL `${host}/artifactory/api/system/licenses/`
+See [CHANGELOG.md](CHANGELOG.md) for full details
 
-You can either access it via api, or web browser - it does require admin level credentials, but it's one of the few APIs that will work without a license (side node: you can also install your license here with a `POST`)
-```sh
-curl -sL ${host}/artifactory/api/system/licenses/ | jq .
-{
-  "type" : "Enterprise Plus Trial",
-  "validThrough" : "Jan 29, 2022",
-  "licensedTo" : "JFrog Ltd"
-}
-```
+<details><summary>Recent Releases</summary>
+### 6.6.0
```

Comment:
do we need to duplicate information from the changelog here?

Reply:
I'm indifferent. It's nice to not send people to a different page and these are inside a collapsible section. On the other hand, this is another thing for us to sync up during releases.